### DISCUSSION TITLE
Wrong description of the 'pure' parameter of the options() method

### DISCUSSION THREAD

**Discussion by benjamin-hoefer-fnt** [KingSora/OverlayScrollbars#577] [General]
Hi!

I think you've got the wrong description about the options() method inside your readme file.

It is about eventlisteners and not about the options:

<img width="649" alt="image" src="https://github.com/KingSora/OverlayScrollbars/assets/139326200/bb97056c-3758-4b9a-86dd-9f62234a9f12">

Regards
Benjamin

**Comment by KingSora** [comment]
@benjamin-hoefer-fnt Thanks! I'll fix this in the next release.

The correct description is:
```
Whether the options should be reset before the new options are added.
```
